lib/Sema/TypeCheckAttr.cpp

```diff
@@ -286,19 +286,27 @@ class AttributeChecker : public AttributeVisitor<AttributeChecker> {
     if (auto var = dyn_cast<VarDecl>(D)) {
       // @actorIndependent is meaningless on a `let`.
       if (var->isLet()) {
-        diagnoseAndRemoveAttr(attr, diag::actorisolated_let);
+        diagnoseAndRemoveAttr(attr, diag::actorindependent_let);
         return;
       }
 
-      // @actorIndependent can not be applied to stored properties.
+      // @actorIndependent can not be applied to stored properties, unless if
+      // the 'unsafe' option was specified
       if (var->hasStorage()) {
-        diagnoseAndRemoveAttr(attr, diag::actorisolated_mutable_storage);
-        return;
+        switch(attr->getKind()) {
```

Comment:
Nit: llvm-style switches please.

Reply:
What would that look like? I wasn't able to find anything on the web.

Reply:
Huh, that's an interesting oversight in the coding standards then. Anyhow

```
switch (subject) {
case <case>:
  <stmt>
...
}
```

`clang-format` can do this for you.